src/librustdoc/clean/types.rs

```diff
@@ -932,42 +932,37 @@ crate enum DocFragmentKind {
     RawDoc,
 }
 
-// The goal of this function is to apply the `DocFragment` transformations that are required when
-// transforming into the final markdown. So the transformations in here are:
+// The goal of this function is to apply the `DocFragment` transformation that is required when
+// transforming into the final Markdown, which is applying the computed indent to each line in
+// each doc fragment (a `DocFragment` can contain multiple lines in case of `#[doc = ""]`).
 //
-// * Applying the computed indent to each lines in each doc fragment (a `DocFragment` can contain
-//   multiple lines in case of `#[doc = ""]`).
-// * Adding backlines between `DocFragment`s and adding an extra one if required (stored in the
-//   `need_backline` field).
+// Note: remove the trailing newline where appropriate
```

Comment:
While you're rewriting these comments, can you make them doc-comments so they show up on doc.rust-lang.org/rustdoc?